src/Sensors/GPS/MAX_M10S.cpp

```diff
@@ -18,7 +18,7 @@ namespace mmfs
         while (m10s.begin() == false && count < 3) // Connect to the u-blox module using Wire port
         {
             // Serial.println(F("u-blox GNSS not detected at default I2C address. Retrying..."));
-            // TODO: RECORD FAILURE IN RECORDDATA
+            // TODO: RECORD FAILURE IN RECORDDATA?
```

Comment:
yea we should do this. honestly we should prob use the logger on more sensors. thats what it's there for.

Reply:
I agree, though this specific error though will be recorded by the `State` so idk if we need it here

```cpp
bool State::init()
{
            if (sensors[i])
            {
                tryNumSensors++;
                if (sensors[i]->begin())
                {
                    good++;
                    snprintf(logData, 100, "%s [%s] initialized.", sensors[i]->getTypeString(), sensors[i]->getName());
                    logger.recordLogData(INFO_, logData);
                }
                else
                {
                    snprintf(logData, 100, "%s [%s] failed to initialize.", sensors[i]->getTypeString(), sensors[i]->getName());
                    logger.recordLogData(ERROR_, logData);
                }
            }
            else
            {
                snprintf(logData, 100, "A sensor in the array was null!");
                logger.recordLogData(ERROR_, logData);
            }
}
```